publication-commons/src/test/java/cucumber/permissions/file/RoleParser.java

```diff
@@ -0,0 +1,18 @@
+package cucumber.permissions.file;
+
+public class RoleParser {
+
+    private final String useraRole;
```

Comment:
*userRole

Reply:
I think it would be cleaner with enums here, but it will force you to rewrite a lot of test columns :)

Reply:
I guess the columns can be as is but return enums from this class. That would probably be ok. Suggest doing that when I get more stuff here